### ISSUE TITLE
Artifact diffing

### ISSUE THREAD

**Issue by ghost** [riidefi/mkw#12]
When the reconstructed main.dol/StaticR.rel doesn't match the genuine game files,
the verification script prints some generic information like segment size.

What's missing is debugging information about which parts of the binary are actually different, by showing hex or assembly diffs.

**Comment by riidefi** [comment]
Yep. Ideally let's integrate this with diff.py. If we generate DWARF info from our sources we could also point to the exact file/line number where the issue is--perhaps automatically putting a NON_MATCHING there.

**Comment by riidefi** [comment]
Partially solved with diff.py. I would like to take this in the direction outlined by #42.

**Comment by em-eight** [comment]
`ninja diff`
